rotkehlchen/tests/data/mocks/test_eth2/validator_daily_stats/999.json

```diff
@@ -0,0 +1 @@
+{"status":"OK","data":[{"attester_slashings":0,"day":0,"day_end":"2020-12-02T12:00:23Z","day_start":"2020-12-01T12:00:23Z","deposits":0,"deposits_amount":0,"end_balance":32012007402,"end_effective_balance":32000000000,"max_balance":32012007402,"max_effective_balance":32012007402,"min_balance":32000000000,"min_effective_balance":32000000000,"missed_attestations":3,"missed_blocks":0,"missed_sync":0,"orphaned_attestations":0,"orphaned_blocks":0,"orphaned_sync":0,"participated_sync":0,"proposed_blocks":0,"proposer_slashings":0,"start_balance":32000000000,"start_effective_balance":32000000000,"validatorindex":999,"withdrawals":0,"withdrawals_amount":0},{"attester_slashings":0,"day":-1,"day_end":"2020-12-01T12:00:23Z","day_start":"2020-11-30T12:00:23Z","deposits":1,"deposits_amount":32000000000,"end_balance":null,"end_effective_balance":null,"max_balance":null,"max_effective_balance":null,"min_balance":null,"min_effective_balance":null,"missed_attestations":0,"missed_blocks":0,"missed_sync":0,"orphaned_attestations":0,"orphaned_blocks":0,"orphaned_sync":0,"participated_sync":0,"proposed_blocks":0,"proposer_slashings":0,"start_balance":null,"start_effective_balance":null,"validatorindex":999,"withdrawals":0,"withdrawals_amount":0}]}
```

Comment:
Where is this file used?

Reply:
In `test_validator_daily_stats_with_genesis_event`